PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/GenomeMap.java

```diff
@@ -32,8 +31,8 @@
    */
   private static final float LOAD_FACTOR = 0.50f;
   /**
-   * Defines the initial capacity for the hash map that contains the genomes.
-   * This value is simply the default initial capacity.
+   * Defines the initial capacity for the hash map that contains the genomes. This value is simply
+   * the default initial capacity.
```

Comment:
> Please undo these changes. It is convention to put each sentence on a new line in JavaDoc. (helps with maintainability since 1 sentence == 1 line). You should be able to set up your auto formatting to honour existing (manual) line breaks.